server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java

```diff
@@ -123,13 +123,9 @@ private static LogEntry switchVolumes(LogEntry le, List<Pair<Path,Path>> replace
     else
       switchedPath = le.filename;
 
-    ArrayList<String> switchedLogs = new ArrayList<>();
     String switchedLog = switchVolume(le.filename, FileType.WAL, replacements);
     if (switchedLog != null) {
```

Comment:
This doesn't even need to be assigned to a variable... can just check if method return is null.

Reply:
Fixed